static/app/components/overlay.tsx

```diff
@@ -19,12 +39,46 @@ interface AnimatedOverlayProps extends HTMLMotionProps<'div'> {
    * would be the arrow (or tip).
    */
   originPoint?: OriginPoint;
-  /**
-   * Additional style rules for the overlay content.
-   */
-  overlayStyle?: React.CSSProperties | SerializedStyles;
 }
 
+function getOverlayStyles({theme}: {theme: Theme}) {
+  return `
+    max-width: 24rem;
+    position: relative;
+    border-radius: ${theme.borderRadius};
+    background: ${theme.backgroundElevated};
+    box-shadow: 0 0 0 1px ${theme.translucentBorder}, ${theme.dropShadowHeavy};
+    font-size: ${theme.fontSizeMedium};
+
+    margin: ${space(1)} 0;
+
+    /* Override z-index from useOverlayPosition */
+    z-index: ${theme.zIndex.dropdown} !important;
```

Comment:
I don't know that it should always be a `dropdown` though should it?

Reply:
hmmm we could add another zIndex called `overlay`, maybe with value `1000`, and let people optionally override it with higher values if necessary.
https://github.com/getsentry/sentry/blob/master/static/app/utils/theme.tsx#L620-L652

Reply:
I thought I did have to pass a zindex somewhere hmm